src/System.Reflection.TypeExtensions/tests/PropertyInfo/PropertyGetAccessors2.cs

```diff
@@ -1,189 +0,0 @@
-            MethodInfo[] myMethodInfos = myPropertyInfo.GetAccessors(false);
-            Assert.Equal(0, myMethodInfos.Length);
-        }
-
-        // Verify get static public property when NonPublic is set to false.
-        [Fact]
-        public void PosTest5()
-        {
-            Type myType = Type.GetType("System.Reflection.Tests.MyProperty");
-            PropertyInfo myPropertyInfo = myType.GetProperty("Property3", BindingFlags.Public | BindingFlags.Static);
-            MethodInfo[] myMethodInfos = myPropertyInfo.GetAccessors(false);
-            Assert.Equal(2, myMethodInfos.Length);
-            Assert.Equal("System.String get_Property3()", myMethodInfos[0].ToString());
-            Assert.Equal("Void set_Property3(System.String)", myMethodInfos[1].ToString());
-        }
-
-        // Verify get static public property when NonPublic is set to true.
-        [Fact]
-        public void PosTest6()
-        {
-            Type myType = Type.GetType("System.Reflection.Tests.MyProperty");
-            PropertyInfo myPropertyInfo = myType.GetProperty("Property3", BindingFlags.Public | BindingFlags.Static);
-            MethodInfo[] myMethodInfos = myPropertyInfo.GetAccessors(true);
-            Assert.Equal(2, myMethodInfos.Length);
-            Assert.Equal("System.String get_Property3()", myMethodInfos[0].ToString());
-            Assert.Equal("Void set_Property3(System.String)", myMethodInfos[1].ToString());
-        }
-
-        // Verify get static protected property when NonPublic is set to false.
-        [Fact]
-        public void PosTest7()
-        {
-            Type myType = Type.GetType("System.Reflection.Tests.MyProperty");
-            PropertyInfo myPropertyInfo = myType.GetProperty("Property4", BindingFlags.NonPublic | BindingFlags.Static);
-            MethodInfo[] myMethodInfos = myPropertyInfo.GetAccessors(false);
-            Assert.Equal(0, myMethodInfos.Length);
-        }
-
-        // Verify get static protected property when NonPublic is set to true.
-        [Fact]
-        public void PosTest8()
-        {
-            Type myType = Type.GetType("System.Reflection.Tests.MyProperty");
-            PropertyInfo myPropertyInfo = myType.GetProperty("Property4", BindingFlags.NonPublic | BindingFlags.Static);
-            MethodInfo[] myMethodInfos = myPropertyInfo.GetAccessors(true);
-            Assert.Equal(2, myMethodInfos.Length);
-            Assert.Equal("System.String get_Property4()", myMethodInfos[0].ToString());
-            Assert.Equal("Void set_Property4(System.String)", myMethodInfos[1].ToString());
-        }
-
-        // Type.GetType(string, bool) throws and does not throw for error scenarios
-        [Fact]
-        public void NegTest1()
-        {
-            Type notFoundButNotThrown = Type.GetType("MyTypeThatDoesntExist", false);
-            Assert.Throws<TypeLoadException>(() =>
-           {
-               Type.GetType("MyTypeThatDoesntExist", true);
-           });
```

Comment:
This was removed because it's not relevant to this suite?  Is this handled in a test elsewhere?